tsl/test/expected/compression_errors.out

```diff
@@ -3,6 +3,7 @@
 -- LICENSE-TIMESCALE for a copy of the license.
 \set ON_ERROR_STOP 0
 \set VERBOSITY default
+\set ECHO none
```

Comment:
Why are you setting this?

Reply:
without this all the satements loaded from `test_utils.sql` would be listed in the `out` file.

see the first few line of changes in `tsl/test/sql/compression_errors.sql` - it turns back echo after the file is loaded.

Reply:
should this be part test_utils.sql then?

Reply:
Great idea! as this change will affect other tests as well - I'll open a separate PR for that!